pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "threadpoolctl",
     "tqdm",
     "probeinterface>=0.2.16",
+    "huggingface_hub",
```

Comment:
Let's move this to the `full` installation. The base installation is only for the base SI classes

Reply:
This makes sense.

Reply:
Done.

Reply:
@samuelgarcia Here.